MeetingNMusae/MeetingNMusae/Views/CharacterBox.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CharacterBox.swift
+//  MeetingNMusae
+//
+//  Created by 김태호 on 2022/06/11.
+//
+
+import Foundation
+import SwiftUI
+
+struct CharacterBox: View {
+    let width: CGFloat?
+    let height: CGFloat
```

Comment:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/59143479/173189376-9d221e80-8f39-4240-b430-582ed8a6ebc7.png">

위와 같이 frame에 매개변수를 주지 않고도 쓸 수 있도록 아래와 같이 선언하면 좋을 것 같습니다.

```suggestion
    let width: CGFloat? = nil
    let height: CGFloat? = nil
```

<img width="937" alt="스크린샷 2022-06-11 오후 10 12 03" src="https://user-images.githubusercontent.com/59143479/173189442-af3fb988-9838-4612-86ed-e3bfa40fa6c5.png">

width와 height을 nil로 넘겨주면 `.frame()` modifier에서 처리를 해줍니다!!